Sources/Networking/NetworkController.swift

```diff
@@ -98,4 +98,17 @@ extension NetworkController: NetworkRequestPerformer {
             })
             .eraseToAnyPublisher()
     }
+    
+    public func send(_ request: any NetworkRequest, requestBehaviors: [RequestBehavior]) async throws -> NetworkResponse {
+        try await withUnsafeThrowingContinuation { continuation in
```

Comment:
Is there a good reason to not use `withCheckedThrowingContinuation` here instead of `Unsafe`?

Reply:
I realized that `URLSession` has a async `urlSession.data(for: request.urlRequest)` method which we can use to create the network response in the implementation. 
<img width="1272" alt="Screenshot 2023-04-26 at 11 45 40 AM" src="https://user-images.githubusercontent.com/42153710/234629978-0857be06-fabc-4228-9d14-34893d5d13ff.png">
The implementation would become the above

Reply:
I was thinking the same thing, and wrote up this comment: https://github.com/Lickability/Networking/pull/10#discussion_r1178039804

While concise, there are a few things the 2-line implementation doesn’t handle:
- It doesn’t call the `requestBehaviors` `requestWillSend` or `requestDidFinish`
- It can throw an arbitrary error back to clients. We should keep the error type as `NetworkError`, which involves some duplicate processing of the response code and wrapping any caught errors.

That comment adds a full implementation so you can see how verbose/duplicate it becomes.